src/main/java/io/appium/java_client/remote/AppiumCommandExecutor.java

```diff
@@ -241,6 +241,9 @@ public Response execute(Command command) throws WebDriverException {
                 }
             });
         }
+        if (getAdditionalCommands().containsKey(command.getName())) {
+            super.defineCommand(command.getName(), getAdditionalCommands().get(command.getName()));
```

Comment:
should we verify if the defined command does not override an existing one?

Reply:
At server side of plugin, we allow to override the behaviour of an Appium command though the path of command remain same. Keep the client side in line with server, I would like to allow client plugins also to override the existing command in same path with a different behaviour.